pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = ["setuptools",
             "setuptools_scm>=6.2",
             "cython==0.29.34",
-            "oldest-supported-numpy",
+            "numpy>=1.25,<2",
```

Comment:
Is there a reason to have the minimum build version higher than the minimum runtime version? That’s generally the thing that has caused problems in the past and is the reason for `oldest-supported-bumpy`.

Reply:
Numpy 1.25 is the first version that supports backward compatible builds. The release notes explicitly mention that the mechanism makes oldest-supported-numpy not needed